models/iaf_cond_alpha_mc_fixedca.cpp

```diff
@@ -0,0 +1,853 @@
+  //!< Waveform for calcium spike current
```

Comment:
Could you break this extremely long line to more manageable lines, no more than 132 characters wide?